frontend/src/lib/Quiver.ts

```diff
@@ -327,24 +328,35 @@ export class Quiver {
     return JSON.parse(schema)
   }
 
+  /** Get unprocessed column names for data columns. Needed for selecting
+   * data columns when there is are multi-columns. */
+  private static parseRawColumns(schema: Schema): string[] {
+    return (
+      schema.columns
+        .map(columnSchema => columnSchema.field_name)
+        // Filter out all index columns
+        .filter(fieldName => !schema.index_columns.includes(fieldName))
```

Comment:
`parseRawColumns` feels like maybe the wrong name for this function - what about `getRawColumnNames` or `getColumnNames` instead?

Also, will this `filter` work in all cases? Is it possible for an index column to have the same name as a data column? If that's not possible, can you add a comment to the filter statement explaining why, for future readers?

(Also also, a wee nit: should the "fieldName" variable be "columnName" instead?)